sdks/python/apache_beam/runners/common.py

```diff
@@ -477,8 +482,20 @@ def __init__(self,
 
     # Optimize for the common case.
     main_receivers = tagged_receivers[None]
+
+    # TODO[BEAM-3937]:Remove if block after per-element-output-counter released.
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Use the form `"# TODO(BEAM-1111): My comment"`.